### PR TITLE
Feature/currency

### PR DESCRIPTION
**Pull request by binaryoverload**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] All changes have been tested

## Added
Currency Command
Convert Command
Currency API getting

## Changed
Test bots log everything to #flare-test-bot

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/Events.java [hunk 1]
```diff
@@ -73,10 +66,11 @@ public Events(FlareBot bot) {
 
     @Override
     public void onMessageReactionAdd(MessageReactionAddEvent event) {
-        if(!event.getGuild().getSelfMember().hasPermission(event.getTextChannel(), Permission.MESSAGE_READ)) return;
+        if (!event.getGuild().getSelfMember().hasPermission(event.getTextChannel(), Permission.MESSAGE_READ)) return;
         if (!event.getGuild().getId().equals(FlareBot.OFFICIAL_GUILD)) return;
         event.getChannel().getMessageById(event.getMessageId()).queue(message -> {
-            MessageReaction reaction = message.getReactions().stream().filter(r -> r.getEmote().getName().equals(event.getReactionEmote().getName())).findFirst().orElse(null);
+            MessageReaction reaction =
+                    message.getReactions().stream().filter(r -> r.getEmote().getName().equals(event.getReactionEmote().getName())).findFirst().orElse(null);
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/Events.java]: This formatting thing triggers me so hard :D

File: src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+import java.awt.Color;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.Random;
+
+public class ConvertCommand implements Command {
+
+    private Random random = new Random();
+    public static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("0.#################");
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java]: Preferable to keep these separate, different scopes and a static final variable.

File: src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    @Override
+    public String getDescription() {
+        return "Allows conversion of currencies";
+    }
+
+    @Override
+    public String getUsage() {
+        return null;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java]: :blobthinkingwithportals:

File: src/main/java/stream/flarebot/flarebot/commands/secret/GuildCommand.java [hunk 4]
```diff
@@ -59,6 +59,28 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     channel.sendMessage(embedBuilder.build()).queue();
                     return;
                 }
+            } else if (args[0].equalsIgnoreCase("beta")) {
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/secret/GuildCommand.java]: How would beta access be removed?

File: src/main/java/stream/flarebot/flarebot/Events.java (hunk 1 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/Events.java]: It's on a new line so what? xD

File: src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java (hunk 2 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java]: Not sure what you mean...?

File: src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java (hunk 3 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java]: Shhhh

File: src/main/java/stream/flarebot/flarebot/commands/secret/GuildCommand.java (hunk 4 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/commands/secret/GuildCommand.java]: Toggle, will add it now

Comment (WalshyDev) [reply src/main/java/stream/flarebot/flarebot/commands/secret/GuildCommand.java]: 👍

File: src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java (hunk 2 above)
Comment (WalshyDev) [reply src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java]: Just add a space between them

File: src/main/java/stream/flarebot/flarebot/Events.java (hunk 1 above)
Comment (WalshyDev) [reply src/main/java/stream/flarebot/flarebot/Events.java]: Triggers me is all, literally an = then an indented assignment. *sigh* so dumb

File: src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java (hunk 2 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java]: Oh ok xD

**Pull request merged**